core/src/main/java/org/frankframework/scheduler/job/IbisActionJob.java

```diff
@@ -22,25 +22,28 @@
 
 import org.frankframework.configuration.AdapterManager;
 import org.frankframework.configuration.ConfigurationException;
+import org.frankframework.configuration.ConfigurationWarning;
 import org.frankframework.core.Adapter;
 
 import org.frankframework.doc.Mandatory;
 
-import org.frankframework.management.IbisAction;
+import org.frankframework.management.Action;
 import org.frankframework.scheduler.JobDef;
 
 import org.frankframework.util.EnumUtils;
 
+@ConfigurationWarning("IbisActionJob has been renamed to ActionJob")
+@Deprecated(since = "8.2.0", forRemoval = true)
 public class IbisActionJob extends JobDef {
 	private @Getter @Setter AdapterManager adapterManager;
 	private @Getter String configurationName;
 	private @Getter String adapterName;
 	private @Getter String receiverName;
-	private Action jobAction;
-	private IbisAction ibisAction;
+	private AvailableAction jobAction;
+	private Action action;
 
 	// Subset of the IbisAction enum as we do not want to expose all fields.
-	public enum Action {
+	public enum AvailableAction {
```

Comment:
Renamed this inline enum to make it more clear what it is and it doesn't have the same name as the other action enum